client/packages/requisitions/src/RequisitionService.tsx

```diff
@@ -29,6 +30,13 @@ const customerRequisitionLineRoute = RouteBuilder.create(
   .addPart(':itemId')
   .build();
 
+const indicatorLineRoute = RouteBuilder.create(AppRoute.CustomerRequisition)
+  .addPart(':requisitionNumber')
+  .addPart('indicator')
+  .addPart(':programIndicatorCode')
+  .addPart(':programIndicatorLineId')
```

Comment:
Carryover to make the `indicator` and `code` values in breadcrumbs non-clickable...